main.py

```diff
@@ -2,6 +2,26 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [("George", "Klooney"), ("Kianu", "Reaves"),
```

Comment:
move variable assighning to the start of the funtion and split actors by lines (one actor - one line)